node/typings/custom.d.ts

```diff
@@ -191,3 +191,36 @@ export interface DeliveryId {
   courierName: string | null
   quantity: number | null
 }
+
+export interface GetCostCenterType {
+  getCostCenterById: {
+    name: string | null
+    paymentTerms: {
+      id: string | null
+      name: string | null
+    } | null
+    addresses: {
+      addressId: string | null
+      addressType: string | null
+      addressQuery: string | null
+      postalCode: string | null
+      country: string | null
+      receiverName: string | null
+      city: string | null
+      state: string | null
+      street: string | null
+      number: string | null
+      complement: string | null
+      neighborhood: string | null
+      geoCoordinates: string | null
+      reference: string | null
+    }
```

Comment:
nit: Is it possible to reuse the `Address` interface that is defined in this same file above?

Reply:
sure, changed heere, thanks